gtsam/slam/BoundingConstraint.h

```diff
@@ -60,14 +59,23 @@ struct BoundingConstraint1: public NoiseModelFactorN<VALUE> {
   virtual double value(const X& x, OptionalMatrixType H =
       OptionalNone) const = 0;
 
-  /** active when constraint *NOT* met */
-  bool active(const Values& c) const override {
```

Comment:
is active not used? Or is it in base class? Backwards compat...

Reply:
Yes, the previous BoundingConstraint class has this method. it's for backward compatibility.